app/views/sessions/timeout_check.js

```diff
@@ -0,0 +1 @@
+// this gets called if there's no session
```

Comment:
Why are we calling an empty javascript file?

Reply:
I understand it's being called because there is no session, so why not call anything at all if the file is empty?

Reply:
I think the doc string might be misleading.  Lucas, is is more like: "this gets called in the process of checking the user's session timeout"?

Reply:
@daryanypl I removed this file; the JS request to SessionsController now renders a JSON response if there's no session: `render json: { 'timeout_status': 'no_current_user' }, status: 100`